Firestore/core/src/firebase/firestore/remote/watch_stream.cc

```diff
@@ -84,14 +84,12 @@ void WatchStream::NotifyStreamOpen() {
 }
 
 Status WatchStream::NotifyStreamResponse(const grpc::ByteBuffer& message) {
-  MaybeMessage<google_firestore_v1_ListenResponse> maybe_response =
-      watch_serializer_.ParseResponse(message);
-  if (!maybe_response.ok()) {
-    return maybe_response.status();
+  GrpcByteBufferReader grpc_reader{message};
```

Comment:
Optional: it seems unlikely we'll be mixing string readers and gRPC readers in the same method. Consider naming locals like these just `reader` to save clutter.